presto-docs/src/main/sphinx/release/release-349.md

```diff
@@ -0,0 +1,50 @@
+# Release 349 (XX Dec 2020)
+
+## General Changes
+
+* Improve query performance by reducing worker to worker communication overhead. ({issue}`6283`)
+* Improve performance of queries that contain `IS NOT DISTINCT FROM` join predicates. ({issue}`6404`)
+* Fix failure when restricted columns have column masks. ({issue}`6017`)
+* Fix http client distribution JMX metrics. ({issue}`6453`)
```

Comment:
Should this be "Add"?  Did we remove this by accident and add back, or just expose them correctly?